src/Raven.Client/Documents/Queries/TimeSeries/TimeSeriesAggregation.cs

```diff
@@ -27,12 +27,12 @@ public class TimeSeriesRangeAggregation
 
     public enum AggregationType
```

Comment:
Add comment here specifying that the order matters.
And user explicit values.

Reply:
done